src/utils/FindUserById.tsx

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/User';
+
+export const findUserById = (id: number): User => {
+  return usersFromServer.find(user => user.id === id)!;
```

Comment:
[CODE KNOWLEDGE] You are using the `find` method which returns `undefined` if no elements in the array satisfy the provided testing function. However, you are using the non-null assertion operator (`!`) after the `find` method which will throw an error if `undefined` is returned. You should handle the case when `find` returns `undefined`.